.kitchen.yml

```diff
@@ -23,3 +23,8 @@ suites:
     run_list:
       - recipe[apt]
       - recipe[aurora::slave]
+
+  - name: client
+    run_list:
+      - recipe[apt]
```

Comment:
Could you remind me what does this `apt` recipe is supposed to do ? (especially on centos :smile:)